config/routes/check_records.rb

```diff
@@ -13,6 +13,9 @@
   get "/auth/dfe/callback", to: "omniauth_callbacks#dfe"
   post "/auth/developer/callback" => "omniauth_callbacks#dfe_bypass"
 
+  get "/terms-and-conditions", to: "terms_and_conditions#show"
+  patch "/auth/developer/callback" => "terms_and_conditions#update"
```

Comment:
patch path looks incorrect here.

Not sure how this could have worked — would you mind re-testing this manually and posting screenshots in the PR description?